.env.circleci

```diff
@@ -0,0 +1,59 @@
+APP_NAME="That's My BIS"
+APP_ENV=local
+APP_KEY=
+APP_DEBUG=true
+APP_URL=http://localhost
+APP_DISCORD=
+
+LOG_CHANNEL=stack
+
+DB_CONNECTION=mysql
+DB_HOST=mariadb
+DB_PORT=3306
+DB_DATABASE=thatsmybis_1
+DB_USERNAME=my_user
+DB_PASSWORD=my_password
```

Comment:
Are these placeholders or the values that get used in the system? (I have no idea how this works)

If they're getting used, would it be much work to change the values to something a little cleaner? (ie. drop the `_1` from the database and choose values for `my_user`, `my_password`)